README.md

```diff
@@ -13,6 +13,11 @@ Vagrantfiles provided for different projects. The general guideline is that the
 snippets should be generic, and use parameters to create code reuse as far as
 possible.
 
+It is worth mentioning that since we use Vagrant more or less exclusively with 
+VirtualBox as the virtualization provider, some scripts assume that. Most notably
```

Comment:
This might be more clear if the sentence is ordered so it says that we "assume that vbox is used as provider", rather than phrasing it this way?